src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Create.cs

```diff
@@ -80,5 +83,31 @@ public void FileSystemWatcher_Directory_Create_SymLink()
                 ExpectEvent(watcher, WatcherChangeTypes.Created, action, cleanup, symLinkPath);
             }
         }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "On Desktop the message is different")]        
+        public void FileSystemWatcher_Directory_NotExists()
```

Comment:
this test won't work when we start to localize .NET Core. We normally don't assert full error messages. I recommend removing this test and check if the current test scenario tests for both empty string and a not existing path.